tests/test_activities.py

```diff
@@ -1,106 +1,102 @@
-        print(f"no data for {wf.name}")
+    # resp = read_json("%s.json" % (s))
+    fixture_file = f"{s}.json"
+    try:
+        with open(fixtures_dir / fixture_file) as f:
+            resp = json.load(f)
+    except FileNotFoundError:
         return
+
     data = resp[0]
     data['git_url'] = wf.git_repo
     data['version'] = wf.version
-    db[s].delete_many({})
-    db[s].insert_one(data)
+    test_db[s].delete_many({})
+    test_db[s].insert_one(data)
+
+
+def get_updated_fixture(wf, fixtures_dir):
+    """
+    Read the fixture file and update the version and git_url for the
+    fixtures that of the same workflow type.
+    """
+    updated_fixtures = []
+    fixture_file = f"{wf.collection}.json"
+    try:
+        with open(fixtures_dir / fixture_file) as f:
+            fixtures = json.load(f)
+    except FileNotFoundError as e:
+        return []
+    for fix in fixtures:
+        if fix['type'].lower() != wf.type.lower():
+            continue
+        fix['git_url'] = wf.git_repo
+        fix['version'] = wf.version
+        updated_fixtures.append(fix)
+    return updated_fixtures
 
 
-def test_activies(db):
+@mark.parametrize(
+    "workflow_file", ["workflows.yaml", "workflows-mt.yaml"]
+    )
+def test_activies(test_db, workflow_file, config_dir, fixtures_dir):
     """
     Test basic job creation
     """
-    # init_test(db)
-    reset_db(db)
-    wfs = load_workflows("./tests/workflows_test.yaml")
-    load(db, "data_object_set.json", reset=True)
+    metatranscriptome = False
+    if workflow_file == "workflows-mt.yaml":
+        metatranscriptome = True
+
+    reset_db(test_db)
+    load_fixture(test_db, "data_object_set.json")
+    if metatranscriptome:
+        load_fixture(test_db, "omics_processing_set_mt.json", col="omics_processing_set")
+    else:
+        load_fixture(test_db, "omics_processing_set.json")
+
+    wfs = load_workflows(config_dir / workflow_file)
     for wf in wfs:
-        if wf.name in ["Sequencing", "ReadsQC Interleave"]:
+        if not wf.type:
             continue
-        fix_versions(db, wf)
-    acts = load_activities(db, wfs)
+        # TODO: these tests are very sensitive to the exact content of the fixture files - need to be more robust
+        updated_fixtures = get_updated_fixture(wf, fixtures_dir)
+        if updated_fixtures:
+            test_db[wf.collection].delete_many({})
+            test_db[wf.collection].insert_many(updated_fixtures)
+
+    acts = load_activities(test_db, wfs)
+    # sanity check
     assert acts is not None
-    assert len(acts) == 5
-    assert len(acts[0].children) == 1
-    assert acts[0].children[0] == acts[1]
-
-    allowlist = set(["foo"])
-    acts = load_activities(db, wfs, allowlist=allowlist)
-    assert len(acts) == 0
-
-    allowlist = set(["nmdc:omprc-11-nhy4pz43"])
-    acts = load_activities(db, wfs, allowlist=allowlist)
-    assert len(acts) == 5
+    if metatranscriptome:
+        # seq, rqc, assembly, annotation, expression
```

Comment:
not necessarily in this PR but seq needs to be deprecated.

Reply:
My preference would be for another small feature branch - in the interest of getting this PR landed